task/06-conditions-n-loops-tasks.js

```diff
@@ -332,7 +435,76 @@ function isBracketsBalanced(str) {
  *
  */
 function timespanToHumanString(startDate, endDate) {
-    throw new Error('Not implemented');
+    const timeDiff = (endDate - startDate) / 1000;
+
+    const dif = (value) => {
+        return (value > Math.floor(value) + 0.5)
+            ? Math.round(value)
+            : Math.floor(value);
+    };
+
+    const timeEnum = {
```

Comment:
This object is a `Time Constants`. Enums allow you select specific data from set. For example: DeviceState: ON/OFF, ChatType: Private/Public/Global, LayoutZone: First/Second/Third/Fourth.